### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by rmw**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/rmw/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
\ No newline at end of file
+# ReviewNinja-Welcome
+
+**Thanks for letting us show you around!**
+
+What you’re seeing here is a GitHub pull request with the code diffs
+showing by default. Feel free to look around.
+
+Over on the right is a panel for line notes that you can connect to a
+line of code in the diff. You can also switch to the conversation
```
Comment (rmw) [inline ReadMe.md:9]: I am commenting on stuff !resolve

**Comment by rmw** [comment]
I have a comment

File: ReadMe.md (hunk 1 above)
Comment (rmw) [inline ReadMe.md:9]: !resolved

**Pull request merged**